File-Formats-toc.html

```diff
@@ -161,6 +161,9 @@
             <li>
                 <a href="/file-formats/DocIO/Working-with-Find-and-Replace">Working with Find and Replace</a>
             </li>
+            <li>
```

Comment:
Can you please change into sublist item for Find and replace page

Reply:
Addressed